pkg/manifest/helm_processor.go

```diff
@@ -162,6 +172,14 @@ func (h *helm) Uninstall(stringifedManifest string, info types.InstallInfo, tran
 		return false, err
 	}
 
+	for i := range postRuns {
```

Comment:
same here, if the purpose of this `postRun` is add additional customized functions after install/uninstall/consistency check, it should happens at the end, right?
Second questions: we have this `postRun` for all cases (install/uninstall/consistency check), what kind of process is fit for all cases? should we offer each case dedicated `postRun`?